src/main/java/com/revature/services/PersonLogic.java

```diff
@@ -0,0 +1,14 @@
+package com.revature.services;
+
+import java.util.List;
+
+import com.revature.domain.Person;
+
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR') Remove this unused import 'com.revature.repositories.PersonRepository'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.revaturelabs.com/coding_rules#rule_key=squid%3AUselessImportCheck)